ffmpeg/extras.c

```diff
@@ -164,6 +164,9 @@ int lpms_get_codec_info(char *fname, pcodec_info out)
     // instead of returning -1
     ret = GET_CODEC_STREAMS_MISSING;
   }
+  if (ic->duration != AV_NOPTS_VALUE) {  
+    out->dur = ic->duration / AV_TIME_BASE;  
```

Comment:
FYI you might want to configure your editor to highlight trailing whitespace because there is some here

```suggestion
  if (ic->duration != AV_NOPTS_VALUE) {
    out->dur = ic->duration / AV_TIME_BASE;
```